generator/uscore/static_test/access_verify_restricted_test.rb

```diff
@@ -0,0 +1,208 @@
+# frozen_string_literal: true
+
+# NOTE: This is a generated file. Any changes made to this file will be
+#       overwritten when it is regenerated
+
+require_relative '../../../../test/test_helper'
+
+describe Inferno::Sequence::ONCAccessVerifyRestrictedSequence do
```

Comment:
Do we want these program-specific sequences in community?

Reply:
Removed access verify tests